draft-yasskin-httpbis-origin-signed-exchanges-impl.md

```diff
@@ -135,59 +149,63 @@ this validity information for some period of time.
 
 ## The Signature Header ## {#signature-header}
 
-The `Signature` header field conveys a single signature for an exchange,
-accompanied by information about how to determine the authority of and
+The `Signature` header field conveys a list of signatures for an exchange, each
+one accompanied by information about how to determine the authority of and
 refresh that signature. Each signature directly signs the exchange's headers and
 identifies one of those headers that enforces the integrity of the exchange's
 payload.
 
 The `Signature` header is a Structured Header as defined by
-{{I-D.ietf-httpbis-header-structure-02}}. Its value MUST be a list (Section 4.8 of
-{{I-D.ietf-httpbis-header-structure-02}}) of parameterised labels (Section 4.4 of
-{{I-D.ietf-httpbis-header-structure-02}}), and the list MUST contain exactly one
-element.
-
-Each parameterised label MUST have parameters named "sig", "integrity",
-"validityUrl", "date", and "expires". Each parameterised label MUST also have
-"certUrl" and "certSha256" parameters. This
-specification gives no meaning to the label itself, which can be used as a
-human-readable identifier for the signature (see {{parameterised-binary}}). The
-present parameters MUST have the following values:
+{{!I-D.ietf-httpbis-header-structure}}. Its value MUST be a parameterised list
```

Comment:
[tangentially related]
While the header-structure spec's 04->06 diff doesn't affect signed exchange much, we were wondering if the header spec would also apply to "Content-Type" header someday, since it disallows identifier being an `item`.
With the change, `v=b1` is invalid and it has to be `v="b1"`.

When parsing signed exchange, Chrome Canary currently allows both `v=b1` and `v="b1"` so we are good here.
However, we might want to change the Accept header to say `v="b1"`